.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = AI/FuzzyLite
 	url = https://github.com/fuzzylite/fuzzylite.git
 	branch = release
+[submodule "3rdparty/find_steam_game"]
+	path = 3rdparty/find_steam_game
+	url = https://github.com/cxong/find_steam_game
```

Comment:
So yeah I'm strongly against adding submodules unless it's very essential or large libraries.